doc/howtos/booting-and-installation.rst

```diff
@@ -351,14 +356,23 @@ the partition tables are wiped out.
 Running Ostro OS in a VirtualBox\* VM
 ======================================
 
-You can run an Ostro OS image within a VirtualBox virtual machine by using the pre-built ``.vdi`` file found
+You can run an Ostro OS image within a VirtualBox virtual machine by using the pre-built ``.ova`` file found
```

Comment:
@kad can we have "latest" symlinks to all of these so we can point them here? Like latest ova/vdi/dsk.xz